setup_and_run_MBEDTLS_PRIVATE.sh

```diff
@@ -0,0 +1,8 @@
+make clean
+sed -i 's/GENERATE_XML           = NO/GENERATE_XML           = YES/g' doxygen/mbedtls.doxyfile
+scripts/config.py realfull
```

Comment:
[This is why `mbedtls_aes_context` is missing](https://github.com/ARMmbed/mbedtls/issues/4518).

For the purpose of finding private fields, `config.py full` will get us most of the way. It will miss a handful of cases such as whichever ECDH context is not used in this configuration. I think `full` is close enough and we can deal with the few remaining cases manually.